ansible/roles/grafana/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: mkdir for config and data
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: "u=rwx,g=rx,o="
+  with_items:
+    - "{{ grafana_config_path }}"
+    - "{{ grafana_lib_path }}"
```

Comment:
docker processes share UID namespace with "dom0", that's why I have separate UIDs in dom0 for every airflow service. It also requires some `passwd` file patching in the image, but I consider it useful for two reasons:
- `ps`, `top` and alike utils have ability to filter out processes for specific UID. Having all dockerized processes running under uid#1000 (named differently in every container) breaks this pattern.
- `ls -l` does not show anything creepy in dom0